test/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM ruby:2.4.1
+RUN apt-get update && \
+    apt-get install -y jq && \
+    apt-get clean && rm -rf /var/lib/apt/lists/*
+
+RUN { \
```

Comment:
what does the { ... } syntax do? I couldn't find a reference...

Reply:
Use it to capture output from everything in the block.

These are equivalent:
```bash
echo foo > file.txt
echo bar >> file.txt
```

And
```bash
{
    echo foo
    echo bar
} > file.txt
```